lib/mix/lib/mix/tasks/test.ex

```diff
@@ -269,16 +269,25 @@ defmodule Mix.Tasks.Test do
     display_warn_test_pattern(matched_warn_test_files, test_pattern)
 
     case CT.require_and_run(files, matched_test_files, test_paths, opts) do
-      {:ok, %{failures: failures}} ->
+      {:ok, %{excluded: excluded, failures: failures, total: total}} ->
         cover && cover.()
 
+        option_only_present = Keyword.has_key?(opts, :only)
```

Comment:
You can name the variable with a question mark too `option_only_present?`.

Reply:
Thanks for the feedback, changed it.